Readme.md

```diff
@@ -148,6 +166,41 @@ The current lead maintainer is [Douglas Christopher Wilson](https://github.com/d
 
 [List of all contributors](https://github.com/expressjs/express/graphs/contributors)
 
+### Collaborators
+
+* [Jon Church](https://github.com/jonchurch) -
+  **Jon Church** <<me@jonchurch.com>>
+* [Wes Todd](https://github.com/wesleytodd) -
+  **Wes Todd** <<wes@wesleytodd.com>>
+
+### Technical Committee
+
+* [Ulises Gascon](https://github.com/UlisesGascon) -
+  **Ulises Gascon** <<ulisesgascongonzalez@gmail.com>> (he/him)
```

Comment:
Personally I would really prefer if we drop the emails from this. We don't *want* people emailing us. This makes is even more clear what those emails are and I don't think I like that. Not that they are ssecret or anything, but putting them in the readme is asking for it.

Reply:
yes I agree with you I got all this information from github sorry if I created any security weakness

Reply:
No its not really. I just dont think it helps us to put it front and center in the readme. There are all public emails we use for commits, so it is as easy as adding `.patch` onto a commit link to find it. It just "suggests" that emailing maintainers is an alright thing to do (it is not, please never out of the blue email a maintainer asking something. Open an issue)

Reply:
removed emails 🚀

Reply:
thank you very much for your suggestion